cpu/kinetis_common/random_rnga.c

```diff
@@ -55,7 +55,7 @@ int random_read(char *buf, unsigned int num)
         }
     }
 
-    return count;
+    return (int)count;
```

Comment:
Without looking further into this: Isn't the logic of this prototype here somehow broken: parameter for the count is unsigned, but the returning count value is signed (and therefore only half the size for possible values)?